Bottom: f8ea4fd4594a31387a734603e1bee6f1d033e4d9
Top:    fbeab5fd2aeadc7434a2500d164aacb2734eedf8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-06-29 02:42:06 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 97f92a2..21ba540 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2462,7 +2462,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
